report/custom.go

```diff
@@ -178,10 +175,11 @@ func (s *CustomMetricSet) Out(w io.Writer) error {
 
 func (s *CustomMetricSet) Compare(s2 *CustomMetricSet) *DiffCustomMetricSet {
 	d := &DiffCustomMetricSet{
-		Key:  s.Key,
-		Name: s.Name,
-		A:    s,
-		B:    s2,
+		Key:    s.Key,
+		Name:   s.Name,
+		A:      s,
+		B:      s2,
+		report: s.report,
```

Comment:
If s.report is nil, s2.report must be used

ref: https://github.com/k1LoW/octocov/pull/308#discussion_r1461269580

Reply:
This is just the idea.

```go
func (s *CustomMetricSet) report() *Report {
	switch {
		s.A.report != nil:
			return s.A.report
		s.B.report != nil:
			return s.B.report
		default: 
			return &Report{}
	}
}
```